packages/shared-ui/src/styles/variables.css

```diff
@@ -10,7 +10,7 @@
   --cb-border-color: #c7c7c7;
   --cb-secondary-link-color: #59acff;
   --cb-box-color: #f5f5f5;
-  --cb-box-color-hover: #bbbbbb;
+  --cb-box-color-hover: #d9d9d9;
```

Comment:
What is this change about? We use that variable also for inside the auth components, so this will also affect other parts of the component.

Reply:
basically, I wanted to centralize the shade of gray in the components and this is a more commonly used color in figma.